### PR TITLE
Privacy/Terms/Accessibility

### PR DESCRIPTION
**Pull request by maxharrisnet**
- New pages for Privacy Policy, Terms of Use, and Accessibility
- Links added to the footer
- Responsive layout for footer links

### CODE REVIEW

**Comment by maxharrisnet** [comment]
@SolivanLau 

There are a lot of spacing changes in this PR (see about.html, 404.html, etc.). I think our formatters are battling a bit, which is fine. Use the "Hide Whitespace" feature on the "Files Changed" tab to get a cleaner view of relevant changes. 

Screen Shot 2024-01-15 at 5 40 01 AM

Also, if you can identify which rules are being applied (I think maybe something to do with line breaks and closing tags), you can add them to the project's shared `.prettierrc` file.

File: css/global.css [hunk 1]
```diff
@@ -877,6 +877,10 @@ footer nav ul {
 
 footer .policy {
```
Comment (maxharrisnet) [inline css/global.css:878]: Adding some styles to deal with 3 new links in the footer

File: css/policy.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
```
Comment (maxharrisnet) [inline css/policy.css:17]: I pretty much copied all of this from about.css

**Comment by maxharrisnet** [comment]
@SolivanLau this touches every page because I had to add links to the footer. If you could check those, that would be awesome. Other changes on other pages are likely just formatting.

File: accessibility-statement.html [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+	<main>
+		<!-- wrapper -->
+		<div class="wrapper">
+			<!-- About Article -->
+			<article>
+				<h1>Accessibility Statement</h1>
+
+				<h2>Our Commitment to Accessibility</h2>
+
+				<p>If These Lands Could Talk (“ITLCT) is committed to ensuring digital accessibility for people with disabilities. We are continually improving the user experience for everyone and strive for compliance with the Web Content Accessibility Guidelines (WCAG) 2.1, Level AA. ITLCT welcomes comments on how to improve accessibility across our digital properties. Please email us at: <a href="accessibility@iftheselandscouldtalk.org" target="_blank">accessibility@iftheselandscouldtalk.org with questions or suggestions.</p>
```
Comment (SolivanLau) [inline accessibility-statement.html]: Just missing an closing tag for the `<a>`. I would also suggest adding `rel="noopener noreferrer"` just in case

**Review by SolivanLau — APPROVED** [review]
Have made a lot of changes to global footer and new pages in CSS and HTML to keep header and footer consistent

**Pull request merged**